posts/2021-01-14-free-nodejs-hosting.md

```diff
@@ -98,87 +102,72 @@ It also has comprehensive [docs](https://render.com/docs?utm_source=geshan.com.n
+Let's look at how we can deploy our Node.js app on Genezio.
 
-* And click "Continue"
-* You can create your own repo on Github from the screen below:
+#### Deploy Node.js app on Genezio
 
-<img class="center" loading="lazy" src="/images/free-nodejs-hosting/06nodejs-vercel-create-repo.jpg" title="Create Github Repo within Vercel for Node.js free hosting" alt="Create Github Repo within Vercel for Node.js free hosting">
+Given you have [registered](https://app.genez.io/auth/signup) on to Genezio with Github and are logged in, do the following steps:
 
-* After that, select the root as the project’s source code, then click “Continue”
+- Fork the [https://github.com/geshan/nodejs-express-tutorial](https://github.com/geshan/nodejs-express-tutorial) to your GitHub account
+- Go to Genezio [dashboard](https://app.genez.io/new-project)
+- Click on "Import from Github" and
+- You will be asked to connect Genez with GitHub.
+- Once connected with GitHub, search for the forked repo.
 
-<img class="center" loading="lazy" src="/images/free-nodejs-hosting/07nodejs-vercel-project-root.jpg" title="Choose root as the project source code in Vercel" alt="Choose root as the project source code in Vercel">
+<img class="center" loading="lazy" src="/images/free-nodejs-hosting/05genez-import-repo.jpg" title="Link your GitHub with Genez via its app" alt="Link your GitHub with Genez via its app">
 
-* Subsequently, click deploy in the next screen:
+- Once the repo is imported to Genezio, if you have any Environment Variables you can add them and click "Create".
 
-<img class="center" loading="lazy" src="/images/free-nodejs-hosting/08nodejs-vercel-deploy.jpg" title="Deploy the app on Vercel with Deploy button" alt="Deploy the app on Vercel with Deploy button">
+<img class="center" loading="lazy" src="/images/free-nodejs-hosting/05genez-create-repo.jpg" title="Create repo in genezio" alt="Create repo in genezio">
 
-* It will build and deploy the app and you will see a screen like below:
+- Subsequently, Genezio will build and deploy the app for you in a live URL in a couple of minutes.
 
-<img class="center" loading="lazy" src="/images/free-nodejs-hosting/09nodejs-vercel-deployed.jpg" title="The app is deployed on Vercel" alt="The app is deployed on Vercel">
+<img class="center" loading="lazy" src="/images/free-nodejs-hosting/05genez-live-url.jpg" title="Genezio URL" alt="Genzio URL">
 
-* Now, click the “Visit” button, you should see something like the following:
+- To view the working app click on the URL on overview page
 
-<img class="center" loading="lazy" src="/images/free-nodejs-hosting/10nodejs-vercel-app-running.jpg" title="The app is running on Vercel" alt="The app is running on Vercel">
+<img class="center" loading="lazy" src="/images/free-nodejs-hosting/05genez-overview.jpg" title="Genezio Dashboard" alt="Genzio dashboard">
```

Comment:
this one too is very small text, probably resize your browser to like 800 px width and 700 px height then take the screenshots again.